app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -365,9 +365,13 @@ def api_version
       @api_version ||= client.api_version
     end
 
-    def ca_cert_store(cert)
-      return if cert.blank?
-      OpenSSL::X509::Store.new.add_cert(OpenSSL::X509::Certificate.new(cert))
+    def ca_cert_store(certs)
+      return if certs.blank?
+      store = OpenSSL::X509::Store.new
+      certs.split(/(?=-----BEGIN)/).each do |cert|
```

Comment:
It's not a match, it's just split into array, interate over it. There is no point to have `?=` (lookahead) option, is my point. Unless @orrabin explains. I am trying to learn here, no big deal.